flink-end-to-end-tests/test-scripts/test_kubernetes_embedded_job.sh

```diff
@@ -19,6 +19,8 @@
 
 source "$(dirname "$0")"/common_kubernetes.sh
 
+IMAGE_BUILD_RETRIES=3
```

Comment:
What's the reason to use a separate variable that is only used in one place? Might it be better to just move the 3 into the `retry_items` call? ...since we do it for the backoff parameter like that as well. 🤔

Reply:
The reason is that I copy pasted the retry call from somewhere else, where this weird combo of variable and magic number was used.

How about introducing a second variable the magic number? I like these variables just for the sake of self-documentation.
WDYT?